release/command/command.go

```diff
@@ -23,6 +25,60 @@ type output struct {
 	err    error
 }
 
+// CommandArgs contains specific arguments to the deploy command.
+type CommandArgs struct {
+	ReleaseData map[string]string
+	Version     string
+}
+
+func parseReleaseData(value string) (map[string]string, error) {
+	dataStrings := strings.Split(value, "=")
+	if len(dataStrings) == 2 {
```

Comment:
With SplitN we'd still do validation here to validate there was both a key and a value.